services/storage/docker-compose.yml

```diff
@@ -52,6 +55,9 @@ services:
       - ./../../vendor/neofs-cli:/neofs-cli
       - ./healthcheck.sh:/healthcheck.sh
       - ./s04tls.crt:/etc/ssl/certs/s04tls.crt
+      - ../nats/client-cert.pem:/nats.tls.cert
```

Comment:
I'd prefer not to use / for anything. Let's create a separate directory, preferably using same path as normal node setup would do.